main/signers/hot/RingSigner/index.ts

```diff
@@ -83,19 +96,17 @@ class RingSigner extends HotSigner {
   }
 
   // TODO: Encrypt all keys together so that they all get the same password
-  async addKeystore(keystore, keystorePassword, password, cb) {
+  async addKeystore(keystore: any, keystorePassword: string, password: string, cb: ErrorOnlyCallback) {
```

Comment:
would be nice to type keystore here

Reply:
I tried quickly but those types are pretty complex and didn't seem like they were exposed by this library. since this is already working I just decided to defer it for now as I didn't see a ton of value in it for the time, but agreed in general we really shouldn't have any `any`s